packages/network-of-terms-catalog/catalog/datasets/abr.jsonld

```diff
@@ -36,6 +36,13 @@
         {
           "@type": "FindAction",
           "query": "file://catalog/queries/lookup/abr-cht.rq"
+        },
+        {
+          "@type": "Action",
+          "actionApplication": {
+            "@id": "https://reconciliation-api.github.io/specs/latest/",
+            "@type": "SoftwareApplication"
+          }
```

Comment:
Does this make sense as a way to indicate that the Reconciliation API should include this dataset (distribution)? It’s a bit weird that this is an action on the SPARQL endpoint. On the other hand, the SPARQL endpoint is exactly what we’re using to fulfil reconciliation requests.

Reply:
Yes, this makes sense to me. If a source, for whatever reason, provides two SPARQL distributions (i.e. in the Dataset Registry), we can use this model to denote which one must be used for recon.

I'm a little bit unsure about `actionApplication` as predicate of the `Action`. If I understand https://schema.org/actionApplication correctly, it should be used on an `EntryPoint`?